Nodes/modules/puzzle_coap/puzzle_coap.c

```diff
@@ -27,24 +30,50 @@ static ssize_t _puzzle_ready_handler(coap_pkt_t *pdu, uint8_t *buf, size_t len,
 static ssize_t _puzzle_maintainance_handler(coap_pkt_t *pdu, uint8_t *buf, size_t len, void *ctx);
 static ssize_t _puzzle_encoder(const coap_resource_t *resource, char *buf, size_t maxlen, coap_link_encoder_ctx_t *context);
 
+static ssize_t _puzzle_serial_handler(coap_pkt_t *pdu, uint8_t *buf, size_t len, void *ctx);
+
 
 /* CoAP resources. Must be sorted by path (ASCII order). */
 static coap_resource_t _resources[] = {
     { "/node/info", COAP_GET, _puzzle_info_handler, NULL },
     { "/node/maintainance", COAP_PUT, _puzzle_maintainance_handler, NULL },
     { "/node/ready", COAP_PUT, _puzzle_ready_handler, NULL },
+    /*{ "/node/serial", COAP_GET, _puzzle_serial_handler, NULL },*/
```

Comment:
?

Reply:
We haven't debugged yet (or I can't remember) what happens if we expose the same resource twice (over UDP vs DTLS). So I left it as we left it at the Hack'n'ACK.